feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/SessionToolbarBehavior.kt

```diff
@@ -0,0 +1,131 @@
+package io.github.droidkaigi.confsched2019.session.ui.widget
+
+import android.content.Context
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.View
+import android.view.animation.AccelerateDecelerateInterpolator
+import android.widget.LinearLayout
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.appcompat.widget.Toolbar
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import androidx.core.view.ViewCompat
+import androidx.core.view.ViewPropertyAnimatorListener
+import androidx.core.view.children
+import androidx.core.widget.NestedScrollView
+import io.github.droidkaigi.confsched2019.session.R
+
+class SessionToolbarBehavior(
+    val context: Context? = null,
```

Comment:
You can use non null Context by using `requireContext()` in Fragment 🙏

Reply:
I fixed it.